### ISSUE TITLE
Mask loss computation: ground truth mask, target mask and predicted mask

### ISSUE THREAD

**Issue by kuzand** [matterport/Mask_RCNN#2779]
From my understanding:
1) Ground-truth masks all have the size of the input image HxW and are given in a training dataset.
2) Predicted masks have fixed sizes mxm (e.g. 28x28 as in the paper).
3) Target masks are computed as intersections of predicted RoIs and their associated ground-truth mask. So target masks have the size of the predicted RoIs (different sizes in general).

In order to compute the mask loss, the target masks and the predicted masks need to have the same size. To achieve this, do we resize a predicted mxm mask to the corresponding target-mask size or vice versa?
